tox.ini

```diff
@@ -1,8 +1,10 @@
 [tox]
 envlist =
-    py{37,38,39,310}-django32
-    isort
-    black
+    ; py{38,39,310}-django32
+    ; py{38,39,310,311}-django{41,42}
+    py{38}-django{41}
+    ; isort
+    ; black
     ; docs
```

Comment:
Why are isort/black commented out now? Also I wonder why line 5 is included instead of lines 3 and 4?